src/main/java/com/datadoghq/sketch/ddsketch/store/DenseStore.java

```diff
@@ -235,6 +235,16 @@ public double getTotalCount() {
         return totalCount;
     }
 
+    @Override
+    public void forEach(BinAcceptor acceptor) {
+        for (int i = minIndex; i <= maxIndex; i++) {
```

Comment:
If `maxIndex == Integer.MAX_VALUE`, which is allowed, albeit unlikely, this loops infinitely. We could do [this](https://github.com/DataDog/sketches-java/blob/f77d000662f038b2aa9aaeec5348491a07ea955e/src/main/java/com/datadoghq/sketch/ddsketch/store/CollapsingLowestDenseStore.java#L118:L125) or maybe something else better.

Reply:
done